client/lib/plans/features-list.tsx

```diff
@@ -1678,10 +1684,15 @@ export const FEATURES_LIST: FeatureList = {
 	[ FEATURE_STYLE_CUSTOMIZATION ]: {
 		getSlug: () => FEATURE_STYLE_CUSTOMIZATION,
 		getTitle: () => i18n.translate( 'Style customization' ),
+		getCompareTitle: () =>
+			i18n.translate( 'Take control of every font, color and detail of your site' ),
 	},
 	[ FEATURE_SUPPORT_EMAIL ]: {
 		getSlug: () => FEATURE_SUPPORT_EMAIL,
 		getTitle: () => i18n.translate( 'Support via email' ),
+		getIcon: () => <img src={ SupportIcon } alt={ i18n.translate( 'Customer support' ) } />,
+		getAlternativeTitle: () => i18n.translate( 'Customer support' ),
```

Comment:
Mobile view has shorter versions of feature titles, so they fit in better.

Reply:
@claudiucelfilip Can we put in a more suitable name for this like getMobileViewTitle.

Reply:
I'd still keep it as generic as possible because the file has historically been the source of content for different designs. We now need it as a mobile view title, but the next iteration may need it for something else.

I wouldn't have added it if there was another method I could use.

Reply:
Yes but this is extremely confusing for someone looking from the outside. I know I was when I read it initially. Because the first question what pops into my head is, there is already a title... what is an `Alternative Title`. If I was going to change the title slightly I will now be confused, Do I add it as getAlternativeAlternativeTitle for the newer most title.

All of this confusion goes away if the  intent of this title was descriptive in the function. 

If you think we should keep this file for generic strings only and this is not a generic use case. Then semantically this  string should not live in this file at all, it should probably be resolved locally in the component itself.